vnext/JSI.Desktop.UnitTests/JsiRuntimeUnitTests.cpp

```diff
@@ -0,0 +1,1072 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include "JsiRuntimeUnitTests.h"
+
+#include <jsi/jsi.h>
+#include <utilities.h>
+
+#include <gtest/gtest.h>
+
+#include <stdlib.h>
+
+// Note: We cannot use unistd.h here because it is a Unix header.
+//       Instead we use the standard chrono header.
```

Comment:
(nit) not sure if this is useful info for anyone, please remove

Reply:
I want to commit this to Facebook's upstream. Will remove this comment then.